src/views/virtualmachines/extensions.ts

```diff
@@ -67,4 +69,13 @@ export const extensions: EncodedExtension[] = [
     },
     type: 'console.page/resource/list',
   } as EncodedExtension<ResourceListPage>,
+  {
+    properties: {
+      component: {
+        $codeRef: 'VirtualMachineMigration',
+      },
+      path: ['/k8s/ns/:namespace/:kind/:name/migrateStorage'],
```

Comment:
we dont use camel case for url, please change it to hypen migrate-storage or simply migratestorage same as standalone route.